README.rst

```diff
@@ -68,26 +45,9 @@ It will:
 Example of use:
 ~~~~~~~~~~~~~~~
 
-::
-
-    $ dev-clone cofebristol
-
-
-dev-createsuperuser
```

Comment:
Please keep this one in! I am using this command a lot and not using `reset`

Reply:
`make django-dev-createsuperuser` does this on a per project basis, and takes into account of the different auth models.

Or would you still like this version which works on most projects?

Reply:
ahhh sorry, I got confused! We have `make django-dev-createsuperuser` per project basis that's great! I believe you can remove this one from there if no one else has offence about it